viewing_party/party.py

```diff
@@ -0,0 +1,207 @@
+    
+    if len(user_data["watched"]) == 0:
+        return None
+    else:
+        for index in range(len(user_data["watched"])):
+            capitalize_genre = user_data["watched"][index]["genre"].capitalize()
+            if capitalize_genre in most_watch_dict:
+                most_watch_dict[capitalize_genre] += 1
+            else:
+                most_watch_dict[capitalize_genre] = 1
+                most_watch_dict = most_watch_dict 
+        
+        most_watched_list = most_watch_dict.values()
+        # for key, value in most_watch_dict.items():
+        #     most_watched_list.append(value)
+
+        most_watch_genre = max(most_watched_list)
+        for movie, count in most_watch_dict.items():
+            if count == most_watch_genre:
+                most_watched_genre_string = movie
+    return most_watched_genre_string
+
+
+# wave 3/1 Get movies that the user has watched but none of their friends have watched
+
+def store_watched_title_from_friends(user_data):
+    
+    unique_watched_list = []
+    for index in range(len(user_data["friends"])): 
+        for key, value in user_data["friends"][index].items():
+            for movie in value: 
+                unique_watched_list.append((movie["title"]))
+    return (unique_watched_list)
+
+
+def get_unique_watched(user_data):
+    unique_movie_list = []
+    watched_list_from_friends = store_watched_title_from_friends(user_data)
+    
+    for index in range(len(user_data["watched"])):
+        unique_movie_dic = {}
+        if user_data["watched"][index]["title"] not in watched_list_from_friends:
+            unique_movie_dic["title"] = user_data["watched"][index]["title"]
+            unique_movie_list.append(unique_movie_dic) 
+    return unique_movie_list
+   
+
+# Wave 3/2 Get movies which at least on of the user's friends have watched, but user has not watched
+
+def store_user_watched_in_list(user_data):
+    user_watched_list = []
+    for index in range(len(user_data["watched"])):
+        user_watched_list.append(user_data["watched"][index]["title"])
+    return (user_watched_list) 
+
+def get_friends_unique_watched(user_data):
+    unique_movie_list = []
+    friends_watched_list = store_watched_title_from_friends(user_data)
+    user_watched_list = store_user_watched_in_list(user_data)
+    
+    for movie in friends_watched_list:
+        unique_movie_dic = {}
+        if movie not in user_watched_list:
+            unique_movie_dic["title"] = movie
+            if unique_movie_dic not in unique_movie_list: 
+                unique_movie_list.append(unique_movie_dic) 
+    return (unique_movie_list)
+
+
+
+# wave 4 Create a list of recommended movies 
+def store_watched_movie_from_friends(user_data):
+    
+    unique_watched_list = []
+    for index in range(len(user_data["friends"])): 
+        for key, value in user_data["friends"][index].items():
+            for movie in value: 
+                unique_watched_list.append((movie))
```

Comment:
If you're only using the value you can use `values` instead of `items`:

```suggestion
        for movies in user_data["friends"][index].values():
            for movie in movies: 
                unique_watched_list.append((movie))
```